decision_service/rlclientlib/periodic_bg_proc.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+#include <algorithm>
+#include <thread>
+#include "err_constants.h"
+#include "error_callback_fn.h"
+#include "api_status.h"
+#include "interruptable_sleeper.h"
+
+namespace reinforcement_learning {
+  class error_callback_fn;
+}
+
+namespace reinforcement_learning { namespace utility {
+
+  template<typename BGProc>
+  class periodic_bg_proc {
```

Comment:
>periodic_bg_proc [](start = 8, length = 16)

can you rename to periodic_background_proc?

Reply:
Done

---
In reply to: [193488786](https://github.com/JohnLangford/vowpal_wabbit/pull/1495#discussion_r193488786) [](ancestors = 193488786)